nexus/types/src/external_api/views.rs

```diff
@@ -870,8 +870,8 @@ pub struct Group {
 
 /// View of a Built-in User
 ///
-/// A Built-in User is explicitly created as opposed to being derived from an
-/// Identify Provider.
+/// Built-in users are identities internal the system, used when the control
+/// plane performs actions on its own behalf
```

Comment:
```suggestion
/// Built-in users are identities internal to the system, used when the control
/// plane performs actions on its own behalf
```

consider "on its own behalf" -> "autonomously"?